### PR TITLE
Add some missing commas.

### PR DESCRIPTION
**Pull request by corrigentia**

### CODE REVIEW

File: getting_started/step_by_step/intro_to_the_editor_interface.rst [hunk 1]
```diff
@@ -74,17 +74,17 @@ Now you should see this interface, with empty docks on the right side.
...
 **workspaces**, and the **playtest buttons**.
 
-On the left side you have the **FileSystem dock**, where you’ll manage
+On the left side, you have the **FileSystem dock**, where you’ll manage
 your project files and assets.
 
 |image11|
 
-On the right side you’ll find the **Scene dock** that lists the active
+On the right side, you’ll find the **Scene dock**, that lists the active
```
Comment (Calinou) [inline getting_started/step_by_step/intro_to_the_editor_interface.rst]:
```suggestion
On the right side, you’ll find the **Scene dock** which lists the active
```

**Comment by cbscribe** [comment]
I don't think these commas are necessary. Example:

> When the folder path is correct you'll see a green checkmark.
> On the right side you’ll find the **Scene dock**

These are single direct statements. There's no need for a pause or break after "correct" or "side". It adds no clarity to the sentence to insert a comma.

When there's a prepositional phrase at the start of a sentence, it is optional and sometimes preferred (as in this sentence) to add a comma for clarity. This is primarily the case if the prepositional phrase is long (in which case it likely needs to be split into two sentences to avoid a run-on). That isn't the case with these particular cases (although the first one is borderline).

Technical writing should be direct and clear. We also have a lot of non-native speakers, which inclines me to advocate for simple and direct language whenever possible.

**Comment by corrigentia** [comment]
@cbscribe : I despise prevaricating prescriptive grammar and conflicting style guides. Vide xkcd's take on [Competing Standards](https://m.xkcd.com/927/). In my book, a subordinate clause that comes before a main clause should always be separated from the latter by a comma, plain and simple. Even though no pause or break may occur in speech, punctuation like this is used in writing to convey meaning beyond what is orally transmitted. [English Punctuation](https://www.thefreedictionary.com/English-Punctuation.htm): Punctuation refers to the specific markings, signs and symbols that are used in and around sentences to give them structure and to allow for correct understanding and comprehension.
Providing structure
If we think of words as bricks that build a sentence (arranged in a certain pattern according to English grammar), punctuation could be thought of as the mortar that holds the bricks together. Without punctuation, our writing would just be a continuous stream of words that lacked structure, pacing, and, ultimately, meaning. For instance, let’s see how this same paragraph would look without any punctuation:

    if we think of words as bricks that build a sentence arranged in a certain pattern according to English grammar punctuation could be thought of as the mortar that holds the bricks together without punctuation our writing would just be a continuous stream of words that lacked structure pacing and ultimately meaning for instance lets see how this same paragraph would look without any punctuation

While it’s possible to discern some of the meaning, it becomes remarkably difficult to understand what is being said when no punctuation marks are used. Furthermore, I don't know how you verbalize the sentences in question, but I make a point of helping the listener understand what I am trying to say in my elocution. To recapitulate, commas are necessary and important.

**Comment by corrigentia** [comment]
@akien-mga : You're welcome! :stuck_out_tongue: Always nice to be appreciated! 
Seriously now, I'm trying to improve the quality of the documentation for my own potential future use.

**Comment by akien-mga** [comment]
Those changes should be made against the `master` branch, not `stable`.

**Comment by akien-mga** [comment]
> Those changes should be made against the master branch, not stable.

Note: I know that it's the "Edit on GitHub" button when browsing `stable` docs which automatically directs you there, but sadly it's not easily configurable yet (rtfd/readthedocs.org#2971).

**Comment by akien-mga** [comment]
Changing the target branch doesn't work sadly, as the branches have different and incompatible commits (this PR now has 135 commits :)).

**Comment by akien-mga** [comment]
Merged manually as 06ffd18, thanks.

**Comment by akien-mga** [comment]
Merged manually as 06ffd18, thanks.

**Comment by akien-mga** [comment]
Upon re-review, I think that adding those commas doesn't hurt, and most of them seem to make things better structured. So, good enough for me :)

Merged manually in the master branch as 59dc710d.

**Comment by corrigentia** [comment]
> Merged manually [...].

@akien-mga : I'm sorry I'm making you do so much. :blush: 

> Upon re-review, I think that adding those commas doesn't hurt, and most of them seem to make things better structured. So, good enough for me :)

And I appreciate the kind words! :grinning:

**Pull request closed without merging**